### DISCUSSION TITLE
PFD and overlapping area

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6255] [General]
Hi developer team,

I think you know that.. the area in the ND for the VOR (L/R) is overlapping the part for the flightplan a bit to much ...

greets 

![grafik](https://user-images.githubusercontent.com/76531893/143686099-a8aaad72-bdcc-45ff-b665-21ab9dcfae9b.png)

**Comment by tracernz** [comment]
These masks are based on a good source. There a various messages that appear on the spot above the right VOR information.
